runtime-config-linux.md

```diff
@@ -131,6 +131,58 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+The path to the cgroups can to be specified in the Spec via `cgroupsPath`.
+`cgroupsPath` is expected to be relative to the cgroups mount point.
+If not specified, cgroups will be created under '/' with the base name of the bundle.
```

Comment:
On Thu, Sep 10, 2015 at 10:46:59AM -0700, Vish Kannan wrote:
> > +If not specified, cgroups will be created under '/' with the base name of the bundle.
> 
> I'm in favor of defaulting to sub-containers of root.

If anyone disagrees with *that*, I think it's a whole nother can of
worms.  I think the disagreement so far has been about what the
sub-container is going to be called, not where it will be rooted.